test/fixtures/lightscript/tilde-calls/this-expression/actual.js

```diff
@@ -0,0 +1 @@
+a~this()
```

Comment:
Let's add a failing test with a reserved word like `yield`, to ensure it's not later changed to an incorrect impl. Instead of expected.json add an options.json with a "throws" key (check other files for examples )